tool/cargo/Syncer.kt

```diff
@@ -30,23 +30,21 @@ import com.vaticle.dependencies.tool.cargo.Syncer.ShellArgs.BAZEL
 import com.vaticle.dependencies.tool.cargo.Syncer.ShellArgs.BAZEL_BIN
 import com.vaticle.dependencies.tool.cargo.Syncer.ShellArgs.BUILD
 import com.vaticle.dependencies.tool.cargo.Syncer.ShellArgs.INFO
-import com.vaticle.dependencies.tool.cargo.Syncer.ShellArgs.OUTPUT_BASE
-import com.vaticle.dependencies.tool.cargo.Syncer.ShellArgs.OUTPUT_GROUPS_RUST_CARGO_SYNC_PROPERTIES
+import com.vaticle.dependencies.tool.cargo.Syncer.ShellArgs.OUTPUT_GROUPS
```

Comment:
Let's rename this to `//tool/ide` and call the `Syncer` the `ManifestSyncer`.

Once API-rearchitecture merges we'll have to propagate again

Reply:
Renaming to `//tool/ide/RustManifestSynker`